app/controllers/api/v1/employees_controller.rb

```diff
@@ -39,7 +39,15 @@ def update
       def destroy
         authorize! member, with: Employees::UserPolicy
         member.destroy
-        respond_to { |format| format.json { head :no_content } }
+        no_content_response
+      end
+
+      def upload
+        authorize! Object, with: Employees::UserPolicy
+        attachment = params[:employees]
+        no_content = -> { no_content_response }
+        json_content = ->(message, status) { render json: { message: message }, status: status }
+        Employees::UploadingAssistant.call(attachment, company, no_content, json_content)
```

Comment:
I don't like idea to move controller responsibilities to service, I just wanted to have controller which checks on service that everything goes smooth or not and responds, but not returning actual response from service :)

Reply:
I am ok with it. But why we need an additional service? I propose next solution:
```
      def upload
        authorize! Object, with: Employees::UserPolicy
        attachment = params[:employees]
        if attachment.blank?
          no_content_response
        elsif
          answer = upload_checker(attachment)
          render json: { message: answer[:message] }, status: answer[:status]
        end        
      end

private

      def upload_assistant(attachment)        
        if CsvHeadersValidator.call(attachment)
          Delayed::Job.enqueue ImportEmployeesWorker.new(attachment, company)
          { message: I18n.t('employees.upload.started'), status: :accepted }
        else
          { message: I18n.t('employees.upload.invalid_headers'), status: :not_acceptable }
        end
      end      
```

Reply:
It's not bad, but, I'd move upload_assistant to separate class